### PR TITLE
frontpage rework

### PR DESCRIPTION
**Pull request by Raemon**

### CODE REVIEW

**Comment by jimrandomh** [comment]
Icons on PostsItem2 cause it to be variable height, in a way that looks janky, at the breakpoint level where it first goes into two-line format. (Specifically: the curated icon, and the AF Omega button.)

**Comment by jimrandomh** [comment]
Tests run during our pair review session:

 * Horizontal resize test on `Home2`

**Comment by jimrandomh** [comment]
At widest breakpoint, expanding comments produces a gray background that's misaligned (appears to have a right edge, but goes through it.)

**Comment by jimrandomh** [comment]
Subscribe icons row should not wrap. (Manifests only at ~280px, smaller than our smallest supported screen size, but might manifest later.)

**Comment by jimrandomh** [comment]
Section footer generally gets wonky with wrapping at small widths, probably needs an explicit, entirely separate layout for `xs`.

**Comment by jimrandomh** [comment]
Section titles have a cursor which suggests they're clickable, but they aren't.

**Comment by jimrandomh** [comment]
Load More button on Home2 recent discussion gives no feedback when clicked.

**Comment by jimrandomh** [comment]
When Include Personal Blogposts is toggled, the animation gets part-way through and then pauses.

**Comment by jimrandomh** [comment]
When you use the expand-parent button on a comment, you get
```
Warning: Failed prop type: The prop `post` is marked as required in `RecentCommentsItem`, but its value is undefined.
```

**Comment by jimrandomh** [comment]
The label "Include Personal Blogposts" is incorrect; it also includes Meta. Since this redesign is already making it much harder to distinguish front page from other sections, it's pretty important to not say anything false about what sections are being displayed.

**Comment by Raemon** [comment]
transition effect looks weird when you're adjusting the page size

**Comment by Raemon** [comment]
Menu Is Missing from mobile postsItem

**Comment by Raemon** [comment]
Sticky doesn't work, open threads maybe specifically don't work?

File: packages/lesswrong/components/common/LoadMore.jsx [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
+    cursor: 'default',
+    '&:hover': {
+      opacity: 1
+    }
   }
 })
 
 
-const LoadMore = ({ loadMore, count, totalCount, classes }) => {
+const LoadMore = ({ loadMore, count, totalCount, classes, disabled }) => {
```
Comment (Raemon) [inline packages/lesswrong/components/common/LoadMore.jsx]:
```suggestion
const LoadMore = ({ loadMore, count, totalCount, classes, disabled=false }) => {
```

File: packages/lesswrong/components/common/SectionFooter.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        marginLeft: theme.spacing.unit*2,
+        marginRight: theme.spacing.unit*2,
+      },
+      '&:last-child': {
+        '&:after': {
+          content: '""',
+          margin:0,
+        }
+      },
+      [theme.breakpoints.down('xs')]: {
```
Comment (Raemon) [inline packages/lesswrong/components/common/SectionFooter.jsx]: [Jim via Ray's keyboard] Hoist this above the `& > *` and invert it to `up('sm')`

File: packages/lesswrong/components/common/SectionFooter.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    marginBottom: theme.spacing.unit,
+    marginRight: theme.spacing.unit*3.5,
+    marginLeft: theme.spacing.unit,
+    color: theme.palette.text.secondary,
+    [theme.breakpoints.down('xs')]: {
+      flexWrap: "wrap"
+    },
+    '& > *': {
+      height: 30,
+      '&:after': {
```
Comment (Raemon) [inline packages/lesswrong/components/common/SectionFooter.jsx]: [Jim via Ray's keyboard]
```suggestion
      // After each child of root except the last one, a bullet divider. ("Except the last one" below with a :last-child selector).
      '&:after': {
```

File: packages/lesswrong/components/common/SectionTitle.jsx [hunk 4]
```diff
@@ -36,16 +35,16 @@ const styles = (theme) => ({
 })
 class SectionTitle extends PureComponent {
```
Comment (Raemon) [inline packages/lesswrong/components/common/SectionTitle.jsx]: [Jim via Ray's keyboard] Make function style rather than class style

**Pull request merged**